src/hdf5_io.cpp

```diff
@@ -313,6 +313,65 @@ template bool h5_write_buffer(char const* filename, char const* varname, cl_long
 template bool h5_write_buffer(char const* filename, char const* varname, cl_ulong const* data, size_t size);
 
 
+
+// write a buffer to an HDF5 file using compression
+template<typename TYPE>
+bool h5_append_buffer(char const* filename, char const* varname, TYPE const* data, size_t size)
```

Comment:
Right now it is not. The idea behind it was to be able to append data to exisiting HDF5 entires. This could be important to write live data for example the current GPU power to the HDF5 files. Right now everything gets buffered in RAM and dumped at the end. This is just a stub, because for short execution times, RAM buffering is more efficient.

Reply:
I would favor either a TODO note in the code (without implementation) or a full implementation. Right now, someone who reads the code will expect that this function can be used for the purpose of its name.

Reply:
For now, RAM buffering seems fine, so I will remove the stub code and add a ToDo.